bin/package.js

```diff
@@ -134,6 +134,37 @@ function buildLinux (cb) {
   electronPackager(Object.assign({}, all, linux), done.bind(null, cb))
 }
 
+function postDarwinism () {
+  var plist = require('plist')
```

Comment:
Don't know, but figured it should be a conditional require since it's used exclusively for Mac OS X builds.